utils.c

```diff
@@ -893,3 +908,24 @@ label_exec (UNUSED const char *exec_label)
 #endif
   return 0;
 }
+
+/*
+ * Like strerror(), but specialized for a failed mount(2) call.
+ */
+const char *
+mount_strerror (int errsv)
+{
+  switch (errsv)
+    {
+      case ENOSPC:
+        /* "No space left on device" misleads users into thinking there
+         * is some sort of disk-space problem, but mount(2) uses that
+         * errno value to mean something more like "limit exceeded". */
+        return ("Limit exceeded (ENOSPC). "
+                "(Hint: Check that /proc/sys/fs/mount-max is sufficient, "
+                "typically 100000)");
```

Comment:
I was unsure if `typically` here meant that the default value is supposed to be 100k or if instead that was a common suggested large enough number to use.

And just FTR, `proc(5)` says that the former is true. 100k is the default value.